cmake/windows-setup.cmake

```diff
@@ -75,7 +75,7 @@ if(FLB_WINDOWS_DEFAULTS)
   set(FLB_OUT_LIB               Yes)
   set(FLB_OUT_NULL              Yes)
   set(FLB_OUT_FLOWCOUNTER       Yes)
-  set(FLB_OUT_KAFKA              No)
```

Comment:
This is just for my curiosity.
Should we disable for out_kafka plugin building on Windows and specify enabling parameter on configure at Windows container building?

This is because to build out_kafka plugin on Windows, we have to introduce dependent zlib library for our CI tasks.

Reply:
Did we do this anyway now @cosmo0920? Ah no, I think that was just for our commercial offering...